src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -76,6 +84,10 @@ public HttpRequestMessage BuildRequest(FormulaValue[] args)
                             query.Append(HttpUtility.UrlEncode(valueStr));
                             break;
 
+                        case ParameterLocation.Header:
+                            headers.Add(param.Name, valueStr);
```

Comment:
>valueStr [](http://example.com/codeflow?start=52&length=8)

Does this need to be encoded? Hopefully line 106 is handling the encoding for us  - but it would be good to add a test for that which passes in troublesome characters (like newlines). 

Or are there limits on what chars can be passed in a header?

Reply:
This is not encoded, will add a test for newline

(fiddler view)
![image](https://user-images.githubusercontent.com/69138830/172709047-2ad87d7b-3350-4e19-96a0-3a9b785eb53e.png)

Reply:
If I read it properly, newlines are not allowed in headers (per rfc7230)